src/sentry/static/sentry/app/views/settings/organizationDeveloperSettings/index.jsx

```diff
@@ -103,6 +149,14 @@ const SentryAppName = styled('div')`
   margin-bottom: 3px;
 `;
 
+const StyledInstallButton = styled(
+  withTheme(({...props}) => {
+    return <Button {...props}>{t('Install')}</Button>;
+  })
+)`
+  margin-right: 5px;
```

Comment:
Should we use `space()` so that grid units are maintained in the future?

Reply:
Maybe? I'm not familiar with how that works or really what that mean, heh.

Reply:
you can do `import space from 'app/styles/space';` And then use `${space(1)` in your css template.